pyfr/integrators/dual/pseudo/multip.py

```diff
@@ -93,6 +93,15 @@ def _aux_regidx(iself):
                 def ntotiters(iself):
                     return self.npmgcycles
 
+                if l == self._order:
+                    @property
+                    def stepper_nregs(iself):
+                        return len(iself.stepper_coeffs) - 1
+                else:
+                    @property
+                    def stepper_nregs(iself):
+                        return 0
```

Comment:
No need for a property.

Reply:
There are a few stepper_nregs calls without the parentheses. And it gives the following error if I remove the property decorator;
`TypeError: unsupported operand type(s) for +: 'int' and 'method'`

Reply:
What I meant was just say stepper_nregs = 0 further up